modules/ROOT/pages/single-sign-on-with-oidc.adoc

```diff
@@ -325,6 +325,12 @@ com.bonitasoft.level = FINE
 The workaround is to put the parameters as regular URL query parameters. Bonita portal has a mechanism that will convert them to hash parameters if they need to be (this only works since version 7.8.1 of Bonita). +
 For example instead of `<server_URL>/bonita/portal/homepage#?_p=caselistinguser&_pf=2`, use `<server_URL>/bonita/portal/homepage?_p=caselistinguser&_pf=2`
 
+*Symptom:* After logging in on the OIDC provider, you are redirected to Bonita but you get a 403 error. +
+*Problem:* Either Bonita OIDC module does not manage to obtain the access tokens, or you don't have the required rights to access the page (when trying to display a page of an application without having the required profile for example). +
+*Solution:*
+Check the logs and if you see a message `failed to turn code into token`, you are in the first situation. The next lines in the logs should indicate the origin of the problem.  Usually, you will need to update the file *keycloak-oidc.json* to match the configuration of Bonita client declared on the OIDC provide.
```

Comment:
```suggestion
Check the logs and if you see a message `failed to turn code into token`, you are in the first situation. The next lines in the logs should indicate the origin of the problem.  Usually, you will need to update the file *keycloak-oidc.json* to match the configuration of Bonita client declared on the OIDC provider.
```
maybe?

Reply:
> you are in the first situation

What about the second situation?

Reply:
you are right, I though it was obvious but I should right it otherwise it seems weird.

Reply:
done